### PR TITLE
fixed the weird post list after checkin issue

### PR DESCRIPTION
**Pull request by ProCynic**

### CODE REVIEW

**Comment by ProCynic** [comment]
also number of messages in inbox

File: app/controllers/Posts.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public static void makeNewPost(String postContent) {
     m.put("item", p);
     m.put("user", user());
     m.put("currentUser", user());
-    renderTemplate(m);
+    Application.news(null);
```
Comment (amshali) [inline app/controllers/Posts.java]: No. This is gonna break the post in the main page.
It should be renderTemplate(m);

Comment (ProCynic) [inline app/controllers/Posts.java]: glad I didn't merge it then.

Comment (amshali) [inline app/controllers/Posts.java]: can you fix it?

Comment (ProCynic) [inline app/controllers/Posts.java]: 1 sec

Comment (ProCynic) [inline app/controllers/Posts.java]: ok, the checkin redirect is still screwed, but the main post add works now.

**Pull request merged**